src/assets/i18n/en_US.json

```diff
@@ -660,7 +715,7 @@
   "approval.detailspage.order.request_id": "Requisition number",
   "approval.detailspage.order_date.label": "Creation date",
   "approval.detailspage.order_details.heading": "Order details",
-  "approval.detailspage.order_reference_id.label": "Order number",
+  "approval.detailspage.order_reference_id.label": "Order no.",
```

Comment:
Do we need to spell "no." out for accessibility reasons so that screen readers read it aloud correctly?

BTW, in DE and FR, this key has variables in the value. Does EN need to be adjusted, or the other two languages?

Reply:
We added an additional localization key with the full term for sr-only and use the variables now also in the English language.